elasticsearch_dsl/field.py

```diff
@@ -170,6 +170,9 @@ def to_dict(self):
         d = self._mapping.to_dict()
         _, d = d.popitem()
         d["type"] = self.name
+        for k, v in iteritems(self._params):
+            if v not in ({}, []):
+                d[k] = v
```

Comment:
Thanks for the patch! I think this piece should be solved by calling `super().to_dict()` and merging it with `d` (which will actually make the previous line not needed).

Do you think you could update the patch ao i can merge it?

Thanks again!